swagger.yaml

```
@@ -10,27 +10,404 @@ basePath: "/v1"
+        items:
+          type: string
+        example: [ "work" , "jobless" ]
+      links:
+        type: array
+        items:
+          $ref: '#/definitions/Link'
+      meta_description:
+        type: string
+        example: "Estimates of employment… statistics for the UK."
+      related_data:
+        type: array
+        items:
+          $ref: '#/definitions/Link'
+      related_documents:
+        type: array
+        items:
+          $ref: '#/definitions/Link'
+      related_methodologies:
+        type: array
+        items:
+          $ref: '#/definitions/Link'
+      related_methodology_articles:
+        type: array
+        items:
+          $ref: '#/definitions/Link'
+      sections:
+        type: array
+        items:
+          $ref: '#/definitions/ContentSection'
+      summary:
+        $ref: '#/definitions/summary'
+      tables:
+        type: array
+        items:
+          $ref: '#/definitions/FigureSection'
+      topics:
+        type: array
+        items:
+          $ref: '#/definitions/Link'
+
+  Contact:
+    type: object
+    properties:
+      email:
+        type: string
+        example: "joe.bloggs@ons.gov.uk"
+      name:
+        type: string
+        example: "Joe Bloggs"
+      organisation:
+        type: string
+      telephone:
+        type: string
+        example: "+44 (0)845 601 3034"
+
+  ContentSection:
+    type: object
+    properties:
+      title:
+        type: string
+        description: Title of the section
+        example: "Section Title"
+      content:
+        type: string
+        description: Markdown formatted content for the section
+        example: "Some _content_ in [Markdown](https://en.wikipedia.org/wiki/Markdown)"
+
+  FigureSection:
+    type: object
+    properties:
+      filename:
+        type: string
+        description: Filename of the chart / equation / image / table
+        example: "57ec16d6"
+      title:
+        type: string
+        description: Title of the chart / equation / image / table
+        example: "A Short Title"
+      uri:
+        type: string
+        description: URI for the chart / equation / image / table
+        example: "/location/of/item"
+      version:
+        type: string
+        description: Version
+        example: "2"
+
+  Link:
+    type: object
+    properties:
+      index:
+        type: integer
+      title:
+        type: string
+      uri:
+        type: string
+
+  id:
+    type: string
+    description: id of the bulletin
+    example: "uklabourmarket"
+  edition:
+    type: string
+    description: edition of the bulletin
+    example: "November 2019"
+  version:
+    type: string
+    description: Particular version
+    example: "1"
+  itemCount:
+    type: integer
+    description: count of items returned in list
+    example: 25
+
+  nationalStatistic:
+    type: boolean
+    description: Indicates that this bulletin refers to a National Statistic
```

There is a little bit of extra description on the dataset api spec that could be helpful here:

```suggestion
    description: Indicates that this bulletin refers to a National Statistic. These are certified as compliant with the Code of Practice for Official Statistics.
```